containers/datalab/config/ipython.py

```diff
@@ -28,3 +28,7 @@
 
 # Enable matplotlib renderings to show up inline in the notebook.
 c.InteractiveShellApp.matplotlib = 'inline'
+
+# Customize the Save as HTML handling
+c.TemplateExporter.template_path = ['/datalab/nbconvert']
+c.TemplateExporter.template_file = 'as_html'
```

Comment:
I can check again but IIRC this is actually the important one; the config/nbconvert.py only affects running nbconvert from the CLI if you shell into the container (which I did for testing).